tests/console/start_systemd_testkit_offline.pm

```diff
@@ -118,10 +119,4 @@ sub parse_results_from_output {
     assert_script_run "echo '$json' > /tmp/$results_file";
 }
 
-sub upload_systemdlib_tests_logs {
-    my ($self) = @_;
-    my $out = script_output('journalctl --no-pager -axb -o short-precise');
-    record_info("JOURNAL", "$out");
-}
```

Comment:
This is actually still needed, you will need to add: 

```
my $filename= "my-amazing-test";
log2serial($out, $filename);
```

Reply:
yes, updated now, thanks!